src/perform_step/exponential_rk_perform_step.jl

```diff
@@ -646,6 +646,94 @@ function perform_step!(integrator, cache::HochOst4Cache, repeat_step=false)
   # integrator.k is automatically set due to aliasing
 end
 
+function perform_step!(integrator, cache::ETD2RK4ConstantCache, repeat_step=false)
+
+  #Draft code for new ETD2RK4 integrator
+  #Variable names follow those in Krogstad, 2005
+
+  @unpack t,dt,uprev,uprev2,f,p = integrator
+  A = isa(f, SplitFunction) ? f.f1.f : calc_J(integrator, cache) # get linear operator
+  alg = unwrap_alg(integrator, true)
+  #F1 = integrator.fsalfirst #not sure if FSAL, so ignore for now
+  halfdt = dt/2
+
+  if integrator.iter == 1
+    # Initialize the first step using Dorm-Prince-5 (also 4th-order)
+    aij = [
```

Comment:
We normally write the tableau in https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/src/tableaus/low_order_rk_tableaus.jl so it is only initialized once.